consensus/types/presets/mainnet/phase0.yaml

```diff
@@ -85,4 +85,4 @@ MAX_ATTESTATIONS: 128
 # 2**4 (= 16)
 MAX_DEPOSITS: 16
 # 2**4 (= 16)
-MAX_VOLUNTARY_EXITS: 16
+MAX_VOLUNTARY_EXITS: 16
```

Comment:
Note for reviewers: the contents of the presets and configs are copied verbatim from `consensus-specs`, which is why there are a mix of substantive changes and formatting changes.